### ISSUE TITLE
dockerize the application

### ISSUE THREAD

**Issue by moonpatel** [Peacexoom/AnimeHub#12]
Setup dockerfile for both frontend and backend seperately. Frontend image should contain only react. Backend image should contain both the NodeJS server and MySQL server. As a follow up you can also add docker-compose files to manage the containers.

**Comment by Mithilesh-create** [comment]
I would like to work on this issue . Assign this task to me.

**Comment by Peacexoom** [comment]
go ahead @Mithilesh-create

**Comment by mananbhimjiyani** [comment]
I would like work on it. Please assign this task to me.

**Comment by moonpatel** [comment]
Yeah sure go ahead @mananbhimjiyani ! This issue is already assigned to @Mithilesh-create so whoever submits a PR first will be accepted.

**Comment by Vaibhav-Thalanki** [comment]
Hey I came across this issue, Could you also add me to the list of assignees? I understand I will need to submit a PR first to get accepted. Thank You

**Comment by Peacexoom** [comment]
Go ahead @Vaibhav-Thalanki

**Comment by MeenuyD** [comment]
Hello i would like to work on this can you please assign it to me

**Comment by MeenuyD** [comment]
Hello I would like to work on this issue can you please assign it to me i am ready to create pr
